LEAF_Request_Portal/sources/FormWorkflow.php

```diff
@@ -875,7 +972,7 @@ public function handleAction(int $dependencyID, string $actionType, string $comm
       * @param string $userName Username
       * @return string
       */
-    public function getEmpUIDByUserName(string $userName): string
+    public function getEmpUIDByUserName(string $userName): string|null

```

Comment:
For future reference, this can also be notated like this: 
```suggestion
    public function getEmpUIDByUserName(string $userName): ?string

```